pyblazing/pyblazing/apiv2/context.py

```diff
@@ -762,6 +914,33 @@ def add_remove_table(self, tableName, addTable, table=None):
             self.lock.release()
 
     def create_table(self, table_name, input, **kwargs):
+        """
+        Create a BlazingSQL table.
+        
+        Parameters
+        ----------
+        
+        table_name : string of table name.
+        input : data source for table.
```

Comment:
Mention here in the input what different types of input you can have:
cudf Dataframe, dask_cudf DataFrame, pandas DataFrame, filepath for orc, parquet, csv

Reply:
https://github.com/gumdropsteve/blazingsql/commit/948e0f8f5a0dc684715830af12c2356286327f7f#diff-b733e955c1d24a9bd95af9c95b18d70fR931